src/main/java/uk/gov/hmcts/darts/cases/controller/CaseController.java

```diff
@@ -129,8 +129,8 @@ public ResponseEntity<SingleCase> casesCaseIdPatch(Integer caseId, PatchRequestO
     @Override
     @SecurityRequirement(name = SECURITY_SCHEMES_BEARER_AUTH)
     @Authorisation(contextId = CASE_ID,
-        securityRoles = {JUDGE, ADMIN, REQUESTER, APPROVER, TRANSCRIBER},
-        globalAccessSecurityRoles = {JUDGE, ADMIN})
+        securityRoles = {JUDGE, ADMIN, REQUESTER, APPROVER, TRANSCRIBER, RCJ_APPEALS},
+        globalAccessSecurityRoles = {JUDGE, ADMIN, RCJ_APPEALS})
```

Comment:
is this needed for 'View Case File - All Transcripts Tab'?

Reply:
hi Chris according to the spreadsheet yes, see image. Let me know if I'm mistaken

<img width="1372" alt="Screenshot 2024-03-01 at 10 38 41" src="https://github.com/hmcts/darts-api/assets/9882391/586ff21f-189e-42c6-937d-6696407d1bf0">

Reply:
also from the new conf page created by Leanne with the description of the roles, RCJ Appeals "can view existing transcripts."

Reply:
Thats fine, think i thought you were removing it.